tests/framework/extensions/hardening/k3s/harden_nodes.go

```diff
@@ -0,0 +1,79 @@
+package hardening
+
+import (
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/pkg/nodes"
+	"github.com/sirupsen/logrus"
+)
+
+func CheckClusterHardening(client *rancher.Client, hardened bool, nodes []*nodes.Node, nodeRoles []string) error {
```

Comment:
I'm not sure this naming makes sense. We're not necessary checking, but rather hardening the cluster. Correct?